src/core.ts

```diff
@@ -34,6 +35,7 @@ smoothscroll.polyfill();
     StachePageHeaderModule,
     StacheLayoutModule,
     StacheWrapperModule
-  ]
+  ],
+  providers: [ WindowRef ]
```

Comment:
Add the `WindowRef` provider to the individual modules that require it, instead of the root module (e.g., Back to Top).